From cddb83a1b397c49eea1f72b7b3dc1d18b73773ff Mon Sep 17 00:00:00 2001
From: Felipe Cardozo <fraifelipe@gmail.com>
Date: Sat, 25 Oct 2025 23:31:06 -0300
Subject: [PATCH 020/133] docs: fix handle_error_with_catch_block typo
X-Developer-Signature: v=1; a=openpgp-sha256; l=1141; i=xtex@astrafall.org;
 h=from:subject; bh=Isb+M682M7N7lYEk2v+O5BZs6JT/Z56e6B0qIUcznfg=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn7cn+Xsp319nwq+9th4vaG52cmWEY/atvUs2SHVYH+5
 v2bvFfadUxkYRDjYrAUU2QpMmzwZtVJ5xddVi4LM4eVCWSItEgDAwMDAwsDX25iXqmRjpGeqbah
 nqGhjpGOEQMXpwBMdY4ow3+/ih1SXk9nZvIe1qoOynI7dp5pWsmc5vly3kXORQu1iywZGTbytGz
 /WXBjU+z9IuOznDx8X9uPRz4Im++2obNiZibLFlYA
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit 0ec45050e0e5f503ebd35473e797ab73f8e64f09)
---
 doc/langref.html.in                                             | 2 +-
 ...th_catch_block.zig.zig => handle_error_with_catch_block.zig} | 0
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename doc/langref/{handle_error_with_catch_block.zig.zig => handle_error_with_catch_block.zig} (100%)

diff --git a/doc/langref.html.in b/doc/langref.html.in
index e231a4e9458b..9c54fba920c7 100644
--- a/doc/langref.html.in
+++ b/doc/langref.html.in
@@ -3016,7 +3016,7 @@ or
       {#syntax#}catch{#endsyntax#} after performing some logic, you
       can combine {#syntax#}catch{#endsyntax#} with named {#link|Blocks#}:
       </p>
-      {#code|handle_error_with_catch_block.zig.zig#}
+      {#code|handle_error_with_catch_block.zig#}
 
       {#header_close#}
       {#header_open|try#}
diff --git a/doc/langref/handle_error_with_catch_block.zig.zig b/doc/langref/handle_error_with_catch_block.zig
similarity index 100%
rename from doc/langref/handle_error_with_catch_block.zig.zig
rename to doc/langref/handle_error_with_catch_block.zig
-- 
2.52.0

